packages/app/obojobo-document-json-parser/src/text/chunk-style-list.js

```diff
@@ -0,0 +1,294 @@
+const StyleType = require('./style-type')
```

Comment:
I think this file is included twice

Reply:
(And some of the others below)

Reply:
I included it multiple times, but in different files. Not sure if I miss something.